src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StringConcat.cs

```diff
@@ -506,5 +712,24 @@ static bool isFieldOfMarshalByRef(BoundExpression expr, CSharpCompilation compil
                 return false;
             }
         }
+
+        private static MethodSymbol? FindSpecificToStringOfStructType(TypeSymbol exprType, MethodSymbol objectToStringMethod)
```

Comment:
>FindSpecificToStringOfStructType [](http://example.com/codeflow?start=37&length=32)

Taking other comments into account, it doesn't look like an extraction of this helper is necessary.